src/commands/layout/list.ts

```diff
@@ -0,0 +1,101 @@
+import { Flags, ux } from "@oclif/core";
+import { AxiosResponse } from "axios";
+
+import * as ApiV1 from "@/lib/api-v1";
+import BaseCommand from "@/lib/base-command";
+import { formatDate } from "@/lib/helpers/date";
+import { merge } from "@/lib/helpers/object";
+import {
+  maybePromptPageAction,
+  pageFlags,
+  paramsForPageAction,
+} from "@/lib/helpers/page";
+import { withSpinner } from "@/lib/helpers/request";
+
+export default class EmailLayoutList extends BaseCommand<
+  typeof EmailLayoutList
+> {
+  static summary = "Display all email layouts for an environment.";
+
```

Comment:
```suggestion
static aliases = ["email-layout:list", "email_layout:list"]
```
Try this, this should work. 

https://oclif.io/docs/aliases/#command-aliases